code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx

```diff
@@ -0,0 +1,14 @@
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import { render, screen } from "@testing-library/react";
+
+describe("PageWrapper", () => {
+  test("should render children", () => {
+    render(
+      <PageWrapper>
+        <div>Test children</div>
+      </PageWrapper>
+    );
+
+    expect(screen.getByText("Test children")).toBeInTheDocument();
+  });
```

Comment:
can create variable with `getByText` option and use it like
`expect(variable).toBeInTheDocument()` for more readability and we'll try to follow this convention for tests

Reply:
Fixed. Thank you